enrolment-ui/src/modules/Applications/ApplicationRequests.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Mostly styling, true changes are the "purge" button and adding the modal+state for it.